README.md

```diff
@@ -103,7 +103,9 @@ separated from the output directory by a colon.
 | `Async` |  `true`/`false` | `true` | Whether to generate asynchronous code |
 | `Sync` |  `true`/`false` | `true` | Whether to generate synchronous code |
 | `TestStubs` |  `true`/`false` | `false` | Whether to generate test stub code |
+| `Implementations` |  `true`/`false` | `true` | Whether to generate imlpementation code |
```

Comment:
nit: move this above `TestStubs`, and elaborate that this is about generating the regular non-test interfaces for using the corresponding service? (Could also mention that this options is mostly useful for generating files that _only_ contain the test stubs, without the corresponding "regular" implementation.) Hope that makes sense, "Implementation" alone is not specific enough IMHO.

Reply:
Updated the text, please take a look.